CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## [Unreleased]
+
+### Fixed
+* Comments get removed for method parameter with attribute [#2585](https://github.com/fsprojects/fantomas/issues/2585)
```

Comment:
Super nitpick but all the other entries have `.` at end of the sentence before the `[#`.
No need to update it, just something I should add to the documentation.